contracts/SnappAuction.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity ^0.5.0;
+
+import "./SnappBase.sol";
+
+
+contract SnappAuction is SnappBase {
+  
+    uint16 public constant AUCTION_BATCH_SIZE = 1000;
+
+    uint public auctionIndex;
+    mapping (uint => PendingBatch) public auctions;
+
+    event SellOrder(
+        uint auctionId, 
+        uint16 slotIndex,
+        uint16 accountId, 
+        uint8 buyToken, 
+        uint8 sellToken, 
+        uint128 buyAmount,
+        uint128 sellAmount
+    );
+
+    constructor () public {
```

Comment:
Do you also have to call the constructor of the inherited contract?

Reply:
Nah, I checked this yesterday. Constructor of other contract only needs to be called if there are parameters to pass. We can trust in this because the unit tests haven't failed... although, we may want to update the migrations and change all the tests to check deposits on the `SnappAuction` This would really prove we don't need the constructor.

Reply:
makes sense